### PR TITLE
[docs] Focus platform-specific instructions

### PR DESCRIPTION
**Pull request by jugglinmike**

### CODE REVIEW

File: docs/running-tests/from-local-system.md
Comment (sideshowbarker) [inline docs/running-tests/from-local-system.md]:
```suggestion
On most UNIX-like systems, you can set up the hosts file with:
```

**Review by sideshowbarker — APPROVED** [review]

File: docs/running-tests/from-local-system.md
Comment (jugglinmike) [reply docs/running-tests/from-local-system.md]: Good catch!

File: docs/running-tests/from-local-system.md
Comment (jgraham) [inline docs/running-tests/from-local-system.md]: I think it's not obvious that the following section doesn't apply if you are running in WSL. Also with WSL 2, which is more like a built-in VM, I kind of wonder how easy it is to test the Windows binary from WSL. Has anoyone tried this. I *assume* part of the value proposition of WSL is that it should work, but verification seems useful.

File: docs/running-tests/from-local-system.md
Comment (jgraham) [inline docs/running-tests/from-local-system.md]: This isn't actually true now in the common case. You only need to do this if you are testing something other than Chrome, Firefox and Servo (and probably other Blink-based browsers would work with the same prefs set we use for Chrome).

Comment (jgraham) [reply docs/running-tests/from-local-system.md]: Oh and it's necessary if you want to use something other than wptrunner e.g. just wpt serve. But there is a hppy path here where you don't need to set this up and we should mention that.

File: docs/running-tests/from-local-system.md
Comment (jgraham) [inline docs/running-tests/from-local-system.md]: This change seems to be repeating the information below.

File: docs/running-tests/from-local-system.md
Comment (jugglinmike) [reply docs/running-tests/from-local-system.md]: The text could look something like this:

    If you expect to [use `wpt serve` directly](#via-the-browser) or test a
    browser other than Firefox or Chrome, then you'll need to set up the test
    domains in your [`hosts`
    file](http://en.wikipedia.org/wiki/Hosts_%28file%29%23Location_in_the_file_system)."

That explanation for the "happy path" is potentially more distracting than just saying, "run this command," even given that it might be unnecessary. My clumsy wording could probably be improved, but readers will still have to skip ahead to learn about the differences of the two test execution methods. More fundamentally, they'll have to remember to revisit the instructions if they later decide to use different features.

File: docs/running-tests/from-local-system.md
Comment (jugglinmike) [reply docs/running-tests/from-local-system.md]: You're right! Removed.

File: docs/running-tests/from-local-system.md
Comment (jgraham) [reply docs/running-tests/from-local-system.md]: This is true. Butnot having to alter your system config to run tests is an important usability win in the cases where it applies. For example exactly the same set of caveats apply when running from a gecko checkout. But because by far the most commin requirement is "run tests in gecko using wptrunner", almost no one has to reconfigure their system and the lack of documentation isn't a problem.

File: docs/running-tests/from-local-system.md
Comment (jugglinmike) [reply docs/running-tests/from-local-system.md]: Hm, I don't know anything about that. I'll see what I can do about setting up a Windows 10 VM locally.

**Comment by jugglinmike** [comment]
@jgraham

> I think it's not obvious that the following section doesn't apply if you are running in WSL.

I've added a few words to make this more clear.

> Also with WSL 2, which is more like a built-in VM, I kind of wonder how easy it is to test the Windows binary from WSL. Has anoyone tried this. I *assume* part of the value proposition of WSL is that it should work, but verification seems useful.

[I initially took this to be something you wanted resolved for this patch.](https://github.com/web-platform-tests/wpt/pull/17085#discussion_r289069408) However, it looks like [WSL2 won't be available until the end of the month](https://devblogs.microsoft.com/commandline/announcing-wsl-2/), and this seems like a new recommendation, anyway. Do you think we can merge this patch without an answer for WSL2?

**Comment by jugglinmike** [comment]
@jgraham Sorry, I dropped the ball on this one. When we left off, [you requested that we tell readers about a potential optimization](https://github.com/web-platform-tests/wpt/pull/17085/files#r288984484). I offered a rewording that I didn't really like, so I'm trying another here:

> (You may skip this step if you do *not* expect to [run the tests by visiting them in your browser]([use `wpt serve` directly](#via-the-browser)) and if you only intend to test Firefox or Chrome.)

I'm sensitive to overwhelming the reader, particularly when it comes to branches in the instructions (and even with the understanding that the branch is a simplification). I'm slightly more comfortable with this new wording for a few reasons:

- a parenthetical that follows the instruction because is potentially more obviously optional
- it avoids referencing `wpt serve` specifically since the command has not been introduced at this point in the instructions

I've also rebased the branch to resolve a trivial conflict. [The prior version is available here.](https://github.com/web-platform-tests/wpt/pull/17085/files#r288984484)

Would you mind taking another look?

**Comment by sideshowbarker** [comment]
@jgraham This seems to be waiting on re-review from you.

**Comment by stephenmcgruer** [comment]
This has significant conflicts with the current version of `from-local-system.md`, and would require a non-trivial rebase. Given jugglinmike does not currently (as far as I know) work in WPT, it's not clear to me that it adds enough value for anyone to pick up - I suggest just closing it.

**Comment by sideshowbarker** [comment]
> it's not clear to me that it adds enough value for anyone to pick up - I suggest just closing it.

I agree and I’m fine personally with this being closed. (But will wait to hear from @jgraham or otherwise let somebody else push the close button on it.)

**Pull request closed without merging**
